CHANGELOG.md

```diff
@@ -1,9 +1,16 @@
 # Changelog
 
-## UNRELEASED
+## 3.42.0 (2023-03-16)
 
 ### Adds
 
+* You can now set `style: table` on inline arrays. It will display the array as a regular HTML table instead of an accordion.
+See the [array field documentation](https://v3.docs.apostrophecms.org/reference/field-types/array.html#settings) for more information.
+* You can now set `draggable: false` on inline arrays. It will disable the drag and drop feature. Useful when the order is not significant.
+See the [array field documentation](https://v3.docs.apostrophecms.org/reference/field-types/array.html#settings) for more information.
+* You can now set the label and icon to display on inline arrays when they are empty.
+See the [array field documentation](https://v3.docs.apostrophecms.org/reference/field-types/array.html#whenEmpty) for more information.
+* We have added a new and improved suggestion UI to relationship fields.
```

Comment:
I removed the link to the docs because I think that was about the "browse" option, which is not new, although it may be new to the docs.